server/chat-commands/core.ts

```diff
@@ -1145,21 +1152,21 @@ export const commands: ChatCommands = {
 
 	restoreplayers(target, room, user) {
 		room = this.requireRoom();
-		if (!room.battle) return this.errorReply("You can only do this in battle rooms.");
-		if (room.rated) return this.errorReply("You can only add a Player to unrated battles.");
+		if (!room.battle) return this.errorReply(this.tr("You can only do this in battle rooms."));
+		if (room.rated) return this.errorReply(this.tr("You can only add a Player to unrated battles."));
 
 		let didSomething = false;
 		if (!room.battle.p1.id && room.battle.p1.name !== 'Player 1') {
 			this.parse(`/addplayer ${room.battle.p1.name}, p1`);
 			didSomething = true;
 		}
-		if (!room.battle.p2.id && room.battle.p2.name !== 'Player 2') {
+		if (!room.battle.p2.id && room.battle.p2.name !== this.tr('Player 2')) {
```

Comment:
I like how "Player 1" isn't translated but "Player 2" is.

(Neither should be translated because `room.battle.p2.name` isn't.)

Reply:
lmao, I did this with a script; I guess I forgot to tell it to skip Player 2. Do you want me to push a fix for this?

Reply:
No need, I already fixed this in #8303 when I refactored this function to support 4-player.

Reply:
If you're wondering, this is the exact line: https://github.com/smogon/pokemon-showdown/pull/8303/files#diff-392164446e6ee53d37f87b67cfca3b7c19e200ce73d08d573991a090738b7369R1396